docusaurus/docs/iOS/02-tutorials/02-audio-room-uikit.mdx

```diff
@@ -0,0 +1,1443 @@
+* The users with id `john_smith` and `jane_doe` are added as members to the call
+* And we set the `title` and `description` custom field on the call object
+
+To actually run this sample we need a valid user token. The user token is typically generated by your server side API.
+When a user logs in to your app you return the user token that gives them access to the call.
+To make this tutorial easier to follow we'll generate a user token for you:
+
+Please update **REPLACE_WITH_USER_ID**, **REPLACE_WITH_TOKEN** and **REPLACE_WITH_CALL_ID** with the actual values shown below:
+
+<TokenSnippet sampleApp='audio-rooms' />
+
+With valid credentials in place, we can join the call. When you run the app you'll see the following:
+
+![Preview of the app after step 3](../assets/audioroom_tutorial/basic.png)
+
+### Step 4 - Adding audio room UI elements
+
+In this next step we'll add:
+
+- Room title and description
+- Controls to toggle live mode on/off
+- A list of participants with their speaking status
+
+#### Room Title & Description
+
+Let's create the components we need to render this and add them to the main app view
+
+```swift title=DescriptionView
+final class DescriptionView: UIView {
+
+    struct Content {
+        var title: String
+        var description: String?
+        var participantsCount: Int
+
+        static let empty: Content = .init(title: "", participantsCount: 0)
+    }
+
+    var content: Content = .empty {
+        didSet { updateContent() }
+    }
+
+    lazy var container: UIStackView = .init()
+        .withVerticalAxis()
+        .withoutTranslatesAutoresizingMaskIntoConstraints()
+
+    lazy var titleLabel: UILabel = .init()
+        .withoutTranslatesAutoresizingMaskIntoConstraints()
+
+    lazy var descriptionLabel: UILabel = .init()
+        .withoutTranslatesAutoresizingMaskIntoConstraints()
+
+    lazy var participantsCountLabel: UILabel = .init()
+        .withoutTranslatesAutoresizingMaskIntoConstraints()
+
+    private let call: Call
+    private var cancellables: Set<AnyCancellable> = []
+
+    @available(*, unavailable)
+    required init?(coder: NSCoder) { fatalError("init(coder:) has not been implemented") }
+
+    init(call: Call) {
+        self.call = call
+
+        super.init(frame: .zero)
+
+        addSubview(container)
+        container.pin(to: self)
+
+        container.addArrangedSubview(titleLabel)
+        container.addArrangedSubview(descriptionLabel)
+        container.addArrangedSubview(participantsCountLabel)
+
+        container.spacing = 8
+
+        titleLabel.font = .preferredFont(forTextStyle: .title1)
+        titleLabel.textColor = .label
+
+        descriptionLabel.font = .preferredFont(forTextStyle: .body)
+        descriptionLabel.textColor = .secondaryLabel
+
+        participantsCountLabel.font = .preferredFont(forTextStyle: .caption1)
+        participantsCountLabel.textColor = .secondaryLabel
+
+        call.state
```

Comment:
Same here, some descriptive methods like subscribeToParticipantUpdates would be nice.